packages/wrangler/src/index.tsx

```diff
@@ -1060,12 +1069,18 @@ export async function main(argv: string[]): Promise<void> {
             type: "string",
             describe: "Filter by a text match in console.log messages",
           })
+          .option("ip", {
+            type: "string",
+            describe:
+              'Filter by the IP address the request originates from. Use "self" to filter for your own IP',
+            array: true,
+          })
+          // TODO: is this deprecated now with services / environments / etc?
```

Comment:
I'll have a look while I'm doing the service environments. but I'm honestly not sure how it'll work... I'll have a look.